installer/installer.py

```diff
@@ -51,18 +51,21 @@
 DEFAULT_DAEMON_POLL_INTERVAL = "300s"
 DEFAULT_UNSAFE_SKIP_BACKUP = "true"
 DEFAULT_DAEMON_RESTART_DELAY = "120s"
-
+DAEMON_DOWNLOAD_MUST_HAVE_CHECKSUM="true"
+DAEMON_SHUTDOWN_GRACE="30s"
 
 ###############################################################
 ###     			Systemd configuration      				###
 ###############################################################
 STANDALONE_SERVICE_TEMPLATE = f"https://raw.githubusercontent.com/cheqd/cheqd-node/{DEFAULT_DEBUG_BRANCH}/installer/templates/cheqd-noded.service"
 COSMOVISOR_SERVICE_TEMPLATE = f"https://raw.githubusercontent.com/cheqd/cheqd-node/{DEFAULT_DEBUG_BRANCH}/installer/templates/cheqd-cosmovisor.service"
+COSMOVISOR_CONFIG_TEMPLATE = f"https://raw.githubusercontent.com/cheqd/cheqd-node/{DEFAULT_DEBUG_BRANCH}/installer/templates/cosmovisor-config.toml"
```

Comment:
We probably need to maintain this going forward per SDK and / or Cosmovisor upgrade, correct?

Reply:
We should also be mindful whether configured for mainnet or testnet as well.

Is it worth adding 2 separate recommended configuration templates?

Reply:
We might as well have to document this within our node-docs.

Reply:
We shouldn't expect any differences between mainnet and testnet so far, based on the current configuration options.

In case there's a need for that in the future, we should definitely separate it.